### DISCUSSION TITLE
Moment curvature animation example

### DISCUSSION THREAD

**Discussion by connorferster** [robbievanleeuwen/concrete-properties#56] [General]
Hi @robbievanleeuwen!

Are you able to add the moment curvature animation example you have as an example in the docs?

I introduced concreteproperties in Lesson 06 of my course this week. Students are using it as part of their workbook this week. Folks seem to like it!!!

**Comment by Agent6-6-6** [comment]
@connorferster If you look in the docs at the animation.ipynb file and gen.py. I believe that's what has been used to generate the gif on the front page of the repository. Should give you enough to go on.

I note it does need to be updated to account for the recent addition of the lumped and mesh reinforcement for the `steel_stresses` variable used in `plot_stress_and_graph()` function when looking at the `mk_res_list` results.

**Reply by robbievanleeuwen** [reply]
Yes it was a bit messy so I hid it! @Agent6-6-6 is right, it needs updating, I'll have a crack at it today.

**Reply by robbievanleeuwen** [reply]
Updated in [cbc7620](https://github.com/robbievanleeuwen/concrete-properties/commit/cbc762050acbb581ef47f939713b2a20b4306bfc). Going to keep in out of the examples due to how long it takes to run, feel free to use as you wish!

**Reply by connorferster** [reply]
Thanks so much @Agent6-6-6 and @robbievanleeuwen!
